datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/lineage/mappers/GenericLineageRelationshipMapper.java

```diff
@@ -0,0 +1,46 @@
+package com.linkedin.datahub.graphql.types.lineage.mappers;
+
+import com.linkedin.datahub.graphql.generated.Chart;
+import com.linkedin.datahub.graphql.generated.Dashboard;
+import com.linkedin.datahub.graphql.generated.Dataset;
+import com.linkedin.datahub.graphql.generated.Entity;
+import com.linkedin.datahub.graphql.generated.EntityRelationship;
+import com.linkedin.datahub.graphql.types.common.mappers.AuditStampMapper;
+import com.linkedin.datahub.graphql.types.mappers.ModelMapper;
+
+import javax.annotation.Nonnull;
+
+public class GenericLineageRelationshipMapper implements ModelMapper<com.linkedin.common.lineage.EntityRelationship, EntityRelationship> {
+
+    public static final GenericLineageRelationshipMapper INSTANCE = new GenericLineageRelationshipMapper();
```

Comment:
Anyway to reduce the noise that "Generic" prefix introduces? 

Can we drop "Generic" from these object names?

Reply:
I would prefer to keep these mappers synced with the pdl names